### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by simonv3** [MoveOnOrg/actionkit-templates#5]
This is on Ubuntu 18.04

```
$ pip3 install actionkit-templates
Collecting actionkit-templates
  Could not find a version that satisfies the requirement actionkit-templates (from versions: )
No matching distribution found for actionkit-templates

$ pip install actionkit-templates
Collecting actionkit-templates
  Could not find a version that satisfies the requirement actionkit-templates (from versions: )
No matching distribution found for actionkit-templates
```
But installing it directly from git does work:

```
pip install git+https://github.com/MoveOnOrg/actionkit-templates.git
Collecting git+https://github.com/MoveOnOrg/actionkit-templates.git
  Cloning https://github.com/MoveOnOrg/actionkit-templates.git to /tmp/pip-fc6eDJ-build
Collecting Django==1.8 (from actionkit-templates==0.1.1)
  Downloading https://files.pythonhosted.org/packages/4e/1c/17a429cfb79c1814d1ec31939fc5cf4a8ac68fe934279e095fb6160123a9/Django-1.8-py2.py3-none-any.whl (6.2MB)
    100% |████████████████████████████████| 6.2MB 196kB/s 
Installing collected packages: Django, actionkit-templates
  Running setup.py install for actionkit-templates ... done
Successfully installed Django-1.8 actionkit-templates-0.1.1
```

**Comment by kaganjd** [comment]
I ran into this as well. I just did `python setup.py install` which installed the necessary dependencies as well as actionkit-templates.

Here was my setup with virtualenv:
1. `git clone https://github.com/MoveOnOrg/actionkit-templates`
2. `cd actionkit-templates`
3. Uses Python 2.7, not Python 3, so need to set up a virtualenv with the flag to Python 2.7 location: `virtualenv --python=/usr/bin/python2.7 venv`
4. `source venv/bin/activate`
5. ~~`python setup.py install`~~ `pip install -e git://github.com/MoveOnOrg/actionkit-templates#egg=actionkit-templates`
6. `cd actionkit_templates/templates`
7. `aktemplates runserver`
